### ISSUE TITLE
Some webpage wallpaper stops after focusing on an application for a while

### ISSUE THREAD

**Issue by AL3XPI** [rocksdanister/lively#154]
**Description of the bug**
I use the wallpaper Starfield, and when I focus on another application, such as Chrome, Edge, or Valorant, for a long period of time (30 mins- 1 hr), my wallpaper goes completely white when I head to the desktop and stays that way until I go into Lively and set Starfield as my wallpaper. Another issue I encounter is some lag when I open lively, which is quite odd, and only happens if the wallpaper isn't a white screen and actually plays. I have an RTX2060 and an AMD Ryzen 5 3600, so I doubt that my computer isn't powerful enough for this application.

**To Reproduce**
I have no idea, I redownloaded this application and this still happens. The only setting I have changed is the Video Quality setting to the "1080p+" setting, as I am using a 4K monitor.

**Expected behavior**
I expect the wallpaper to not do this, and unpause itself when I exit out of a game or window, instead of leaving me with a blank, white screen.

**Screenshots**
This is what I mean by a blank white wallpaper:
![Screenshot (37)](https://user-images.githubusercontent.com/35903333/95132801-33d16280-0715-11eb-9f00-8de7639435d2.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro Version 2004
- Wallpaper: Starfield

**Log file**
Share the crashlog file logfile.txt from Local Appdata folder C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs with detailed information (The file gets overwritten each time lively starts, so make sure its from when lively crashed.)
[20201001_062853.txt](https://github.com/rocksdanister/lively/files/5330120/20201001_062853.txt)
[20201003_095040.txt](https://github.com/rocksdanister/lively/files/5330121/20201003_095040.txt)
[20201004_092228.txt](https://github.com/rocksdanister/lively/files/5330122/20201004_092228.txt)
[20201004_142818.txt](https://github.com/rocksdanister/lively/files/5330123/20201004_142818.txt)
[20201005_061816.txt](https://github.com/rocksdanister/lively/files/5330124/20201005_061816.txt)
[20201005_110550.txt](https://github.com/rocksdanister/lively/files/5330125/20201005_110550.txt)

**Comment by AL3XPI** [comment]
@rocksdanister

**Comment by AL3XPI** [comment]
I will try reinstalling Lively, since you all cannot be bothered to address this issue.

**Comment by rocksdanister** [comment]
Seriously? You can't wait a few days.. Lively is being run by one person and I been trying to recreate this issue before commenting.

**Comment by AL3XPI** [comment]
I'm sorry for being so insensitive, I reinstalled Lively and I will keep using the program and tell you if I have any problems, and after today, if there is no blank white screen, I will close the issue.

**Comment by rocksdanister** [comment]
I couldn't recreate it on my end.
My hunch is the gpu driver crashing with the browsers hardware acceleration enabled.
This could also be an issue with some overflowing variable in the webpage(wallpaper) etc..

Lively's crashlog is not very helpful in this case, next time it happens share the crashlog from the chromium browser
`C:\Users\<UserName>\AppData\Local\Lively Wallpaper\Cef\logfile.txt`
(File gets overwritten each time browser is opened, so take a copy before starting a web wallpaper again.)

**Comment by AL3XPI** [comment]
Oh ok, and the white screen has come back. I'll share the file with you.
[logfile.txt](https://github.com/rocksdanister/lively/files/5340751/logfile.txt)

**Comment by rocksdanister** [comment]
Hmm nothing.

1. Try different wallpaper (website type.)
2. Try changing wallpaper pause -> Nothing on fullscreen app in Settings -> Performance

**Comment by AL3XPI** [comment]
When I try to do the second thing, it has some kind of conflict between the windows and I could barely switch in and out of Chrome windows and Edge windows, and when I reverse that change, it goes back to normal, so I'll try using a different wallpaper, even though I really like Starfield.

**Comment by rocksdanister** [comment]
>it has some kind of conflict between the windows and I could barely switch in and out of Chrome windows and Edge windows
makes no sense.. Lively does not make any modifications like that.

I want a list of all the background programs running on your system (system-tray and windows startup.).. I'm vary of things like Valorant anti-cheat which can kill other programs when it feels like(?)

**Comment by AL3XPI** [comment]
![Screenshot (41)](https://user-images.githubusercontent.com/35903333/95335021-b831fb80-0863-11eb-82eb-67fe8dcbb2f1.png)
This is what is usually running on my computer.

**Comment by AL3XPI** [comment]
In my tray, we have Bluetooth, Windows Defender, Edge, Task Manager, Logitech G Hub, Epic Games, Discord, a Battery software, Xbox App, Nvidia Geforce Experience and Control Panel, TaskbarX, Everything by voidtools, and Lively.

**Comment by AL3XPI** [comment]
I am using the Matrix Rain wallpaper and it hasn't done the white screen thing, so far atleast.

**Comment by XuCcc** [comment]
The same issue,i try to pasue and resume it and seems to be useless

**Comment by rocksdanister** [comment]
He has not replied after trying other wallpapers, not sure if his issue is resolved.
Which wallpaper do you have this issue XuCc?

**Comment by AL3XPI** [comment]
I used the matrix rain wallpaper and it has not given me any issues even though it is a website type wallpaper.

**Comment by XuCcc** [comment]
> He has not replied after trying other wallpapers, not sure if his issue is resolved.
> Which wallpaper do you have this issue XuCc?

I use it from a website:

- https://louie.co.nz/25th_hour/
- https://pabilo8.github.io/forest-wallpaper/

**Comment by rocksdanister** [comment]
Does it still happen if you turn off wallpaper pause?
`Lively settings -> Performance -> Application Fullscreen -> Nothing.`

This looks like an issue with the webpage itself, Lively is only minimizing the browser to trigger the browser pause.

**Comment by AL3XPI** [comment]
I will try Starfield with the wallpaper pause off once again. Matrix works even without wallpaper pause off.

**Comment by AL3XPI** [comment]
Starfield is still broken. All I did today was using Chrome and Edge, no fullscreen application. Matrix works fine and it is a webpage wallpaper as well, which means there must be some issue with Starfield.

**Comment by rocksdanister** [comment]
I had a brief look at starfield code, there is a lot of loops etc.. it might be possible to add a browser minimize flag as a fix similar to what I did for Fluid v2 which was having a different issue:
https://github.com/rocksdanister/WebGL-Fluid-Simulation/blob/42daa887c9fc58fc279e4c2ec68dc6836cbde9d8/script.js#L71

Here is a similar Starfield effect:
http://www.chiptune.com/starfield/starfield.html

**Comment by AL3XPI** [comment]
Oh alright, maybe try doing that!

**Comment by rocksdanister** [comment]
Oh I'm just going to remove it from the bundled wallpaper list.. It's just a suggestion if anyone want to try. 

Going to close this for now because it looks like webpage issue, if there is more information its something else will reopen.

**Comment by Fennecai** [comment]
i have this same problem with video wallpapers when using a youtube url
